tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -654,7 +651,7 @@ public List<String> getTopics(Optional<String> topicIncludeList, boolean exclude
         public List<Uuid> getTopicIds(Uuid topicIdIncludeList, boolean excludeInternalTopics) throws ExecutionException, InterruptedException {
             ListTopicsResult allTopics = excludeInternalTopics ? adminClient.listTopics() :
                 adminClient.listTopics(new ListTopicsOptions().listInternal(true));
-            List<Uuid> allTopicIds = null;
+            List<Uuid> allTopicIds;
```

Comment:
Since we assign it just below, could we just do:
```
List<Uuid> allTopicIds = allTopics.listings().get().stream()
        .map(TopicListing::topicId)
        .sorted()
```?